makefile

```diff
@@ -341,7 +348,42 @@ junit-report:
 	rm -f test_reports/junit.xml
 	$(GO_TEST_CMD) bash -c "cat test_reports/*.txt | go-junit-report > test_reports/junit.xml"
 
-proto:
+python: download
```

Comment:
How are the proto dependencies downloaded?

Reply:
We use the tools.go and the other dependencies are already part of go.mod